lib/log_helper.py

```diff
@@ -20,21 +20,35 @@
 
 
 class LogHelper(object):
-    def __init__(self, logfile=None, level=logging.DEBUG, verbose=False):
+    def __init__(self, logfile=None, verbose=False, rotate_size=None):
         self.logfile = logfile
-        self.loglevel = level
 
         self._directory_setup()
 
         logger = logging.getLogger()
-        logging.basicConfig(filename=logfile, level=level)
-        print("Logs available at %s" % logfile)
+        logger.setLevel(logging.DEBUG)
+
+        sh = logging.StreamHandler(sys.stdout)
+        if verbose:
+            sh.setLevel(logging.DEBUG)
+        else:
+            sh.setLevel(logging.INFO)
+        sh.setFormatter(logging.Formatter('%(message)s'))
+        logger.addHandler(sh)
+
+        logger.info("Logs available at %s" % logfile)
+
         # NOTE(maurosr): RotatingFileHandler expects file size in bytes, in
         # short terms we're defining 2MB limit here.
-        logger.addHandler(logging.handlers.RotatingFileHandler(
-            logfile, maxBytes=2 << 20, backupCount=1))
-        if verbose:
-            logger.addHandler(logging.StreamHandler(sys.stdout))
+        if not rotate_size:
+            rotate_size = 2 << 20
```

Comment:
I don't want to lose the comment from line 41 and I think moving it up there would be strange.